http://bugzilla.xfce.org/show_bug.cgi?id=7982

panel-plugin/main.c is using function round, so explicitely link against -lm.  see below.

/path/to/ld: xfce4_diskperf_plugin-main.o: undefined reference to symbol 'round@@GLIBC_2.2.5'
/path/to/ld: note: 'round@@GLIBC_2.2.5' is defined in DSO /lib64/libm.so.6 so try adding it to the linker command line
/lib64/libm.so.6: could not read symbols: Invalid operation
collect2: ld returned 1 exit status

--- panel-plugin/Makefile.am
+++ panel-plugin/Makefile.am
@@ -25,7 +25,8 @@
 
 xfce4_diskperf_plugin_LDADD=					\
 	$(LIBXFCE4PANEL_LIBS)					\
-	$(LIBXFCEGUI4_LIBS)
+	$(LIBXFCEGUI4_LIBS)					\
+	-lm
 
 #
 # Desktop file
